SPEC.md

```diff
@@ -61,17 +61,22 @@ After these a number of fields will exist.
 | Y resolution | 4      | 'Y'   | uint32  | Frame Y resolution (rows)
 | Compression  | 1      | 'C'   | uint8   | Compression scheme in use (0 = uncompressed)
 
+### Optional Header Field
+
+| Name         | Length | Code  | Type    | Description
+| ------------ | ------ | ----- | ------- | ---------------------------------------------
+| Device name  | ???    | 'D'   | string  | Device name e.g. camera model.
```

Comment:
Thanks for updating the spec as well. Maybe say "variable" instead of "???"?

Reply:
Also, the device name isn't the same as the camera model. 

Each of our devices has a unique identifier (e.g. "akaroa03"). That's what goes into this field. The camera model would be a different thing (i.e. the thermal camera hardware model)